src/Suave/Authentication.fs

```diff
@@ -98,8 +99,15 @@ let authenticated relativeExpiry secure : WebPart =
                  (fun _ -> Choice1Of2 data)
                  succeed)
 
-//  let deauthenticate : WebPart =
-//    Cookies.unset_cookies
+let deauthenticate _ : WebPart =
+  unsetPair SessionAuthCookie 
+  >=> unsetPair StateCookie
+
+let deauthenticateWithLogin loginPage : WebPart =
+  deauthenticate()
```

Comment:
I think you can drop the `()` and the `_` from `deauthenticate` as a WebPart is a function to start with.